docs/user-guide/index.md

```diff
@@ -0,0 +1,45 @@
+# Welcome to SciCat Users Guide
+
+This is a short guide to most common questions users of SciCat can face. 
+First of all: What is SciCat? And what are its features, how can I use it? 
+
```

Comment:
Suggestion:
Scicat, a _scientific metadata catalogue_, allows your to explore your datasets through their metadata. SciCat has mechanisms to interact with the related datasets through its flexible integration with most storage systems.